docs/usage/self-hosted-configuration.md

```diff
@@ -63,6 +63,41 @@ But before you disable templating completely, try the `allowedPostUpgradeCommand
 
 ## allowScripts
 
+## allowedHeader
+
+This is option particularly useful when a registry employs a specific authentication system not already covered by Renovate's standard credential handling in `hostRules`.
+By default, all header starting with "X-" are allowed, but you can permit additional header using this option.
+If declared, it will override the default "X-" allowed header, so you should include them in your config if you wish for them to remain allowed.
+`allowedHeader` is an array of minimatch-compatible globs or re2-compatible regex strings.
+
+Examples:
+
+- `/X/` - any header with `X` anywhere in the name (regex)
+- `!/X/` - any header without `X` anywhere in the name (regex)
+- `X-*` - any header starting with `X-` (glob pattern)
+- `X` - only the header matching exactly `X` (exact match glob)
```

Comment:
Please convert this to a table:

| Example header | Kind of pattern | Explanation |
| --- | --- | --- |
| `/X/` | Regex | Any header with `x` anywhere in the name |

And so on.